client/ayon_flame/plugins/create/create_shot_clip.py

```diff
@@ -58,250 +570,294 @@ def process(self):
 
         sorted_selected_segments.extend(unsorted_selected_segments)
 
-        kwargs = {
-            "log": self.log,
-            "ui_inputs": results_back,
-            "basicProductData": self.data,
-            "productType": self.data["productType"]
+        # detect enabled creators for review, plate and audio
+        all_creators = {
+            "io.ayon.creators.flame.shot": True,
+            "io.ayon.creators.flame.plate": True,
+            "io.ayon.creators.flame.audio": pre_create_data.get("export_audio", False),
         }
+        enabled_creators = tuple(cre for cre, enabled in all_creators.items() if enabled)
+
+        instances = []
+
+        for idx, segment in enumerate(sorted_selected_segments):
+
+            clip_index = str(uuid.uuid4())
+            segment_instance_data = instance_data.copy()
```

Comment:
`.copy()` on dictionary is the same as `dict(instance_data)` which does not create copy of value objects (e.g. list)
```suggestion
            segment_instance_data = copy.deepcopy(instance_data)
```